.devcontainer/Dockerfile

```diff
@@ -172,4 +172,6 @@ RUN GOBIN="/usr/bin/" go install -v golang.org/x/tools/gopls@v0.8.3 && \
 #### Update shared library configuration
 RUN ldconfig -v
 
+RUN echo 'alias magma-build-agw="bazel build //... --build_tag_filters=service,util_script,-mme_oai"' >> ~/.bashrc
```

Comment:
This is set for the same user in devcontainer and bazel base, right? Then it should be sufficient to have this is in the bazel base container.